test/unit/modules/syntax.js

```diff
@@ -287,6 +287,17 @@ describe('Syntax', function () {
       );
     });
 
+    it('should not call selection change if editor is not focused to not refocus (T1233316)', function () {
```

Comment:
it's absolutely impossible for me to undestand this, but let keep at it is since test is simple

Reply:
That's because u don't know that `setSelection` triggers focus)
Understand, it was also unexpectedly to me, but that how it works